torch_frame/data/mapper.py

```diff
@@ -341,15 +343,19 @@ def forward(
                     device)
             return feat_dict
 
-        tokenized_list: TextTokenizationOutputs = []
+        tokenized_outputs: TextTokenizationOutputs = []
         for i in tqdm(range(0, len(ser_list), self.batch_size),
                       desc="Tokenizing texts in mini-batch"):
             tokenized_batch: TextTokenizationOutputs = self.text_tokenizer(
                 ser_list[i:i + self.batch_size])
-            tokenized_list.extend(tokenized_batch)
-        for key in tokenized_list[0]:
+            if isinstance(tokenized_batch, Mapping):
+                tokenized_outputs.append(tokenized_batch)
+            else:
+                tokenized_outputs.extend(tokenized_batch)
+
+        for key in tokenized_outputs[0]:
             xs = []
-            for item in tokenized_list:
+            for item in tokenized_outputs:
```

Comment:
Can we branch 
```python
                if item[key].ndim == 1:
                    xs.append([item[key]])
                elif item[key].ndim == 2:
                    for row in item[key]:
                        xs.append([row])
```
based on 
```python
if isinstance(tokenized_batch, Mapping):
```
and then assert on ndim?

Reply:
Do you mean if the `tokenized_batch` is `Mapping`, tensors should be dim=2?